tests/scripts/all.sh

```diff
@@ -1158,6 +1163,7 @@ post_report () {
 
 run_all_components () {
```

Comment:
On the topic of how to list components: unfortunately, there is no way to list all functions in the sh language. I thought of two approaches:

* Grep for function definitions, the way you did it.
* Use bash instead of sh, and `declare -f`. Bash is installed by default on non-embedded Linux and on OSX, but not on FreeBSD.